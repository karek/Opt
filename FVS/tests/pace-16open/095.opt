8
56 47 37 5 51 43 22 44 
