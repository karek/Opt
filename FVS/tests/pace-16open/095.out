8
47 53 37 56 5 51 43 22 
